mail_activity_done/models/mail_activity.py

```diff
@@ -16,7 +16,7 @@ class MailActivity(models.Model):
         compute="_compute_state",
         search="_search_state",
     )
-    date_done = fields.Date("Completed Date", index="btree", readonly=True)
+    date_done = fields.Date("Completed Date", index="btree")
```

Comment:
Why are you removing `readonly=True` here and add it to the view?

Reply:
From the [Migration to version 17.0](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0) guidelines: 

> If a field is marked as readonly=True on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the `readonly` attribute on the views instead.